django/santropolFeast/member/models.py

```diff
@@ -159,12 +159,25 @@ class Client(models.Model):
         (DECEASED, _('deceased')),
     )
 
+    FACTURATION_TYPE = (
+        ('default', _('default')),
+        ('low income', _('low income')),
+        ('solidary', _('solidary')),
+    )
+
     class Meta:
         verbose_name_plural = _('clients')
 
     billing_address = models.ForeignKey(
         'member.Address',
-        verbose_name=_('billing_Address')
+        verbose_name=_('Billing Address')
```

Comment:
Be carefull we do not write in English but we just create id which will then be translated.
Here ```billing_address``` is the good code